CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+## Contributing In General
+
+Our project welcomes external contributions! If you have an itch, please
+feel free to scratch it.
+
+To contribute code or documentation, please submit a pull request to the [GitHub
+repository](https://github.com/IBM/shopping-list-polymer-pouchdb).
```

Comment:
The repo link here is incorrect.

Reply:
I changed all the links from IBM to ibm-watson-data-lab